extension/lib/shopgate/Cart.js

```diff
@@ -41,48 +41,6 @@ class ShopgateCart {
   createItemBuilder (id, quantity) {
     return new ShopgateCartItemBuilder(id, quantity)
   }
-
-  /**
-   * @return {ShopgateCartFlags}
-   */
-  get flags () {
```

Comment:
I believe on changes like this we should first talk. This project was written in this style as a convention for having "private" properties with underscore and having getters that are readonly properties.

We may chose to change this, and if we do, we should we should then change every similar usage to have consistent code style.

Reply:
OK, so I say let's change this, cause all this "style" did was generate a lot of code that doesn't really do anything and doesn't give us any benefit.
I didn't see the need to discuss it cause I thought it's 100% obvious that this code is redundant.

Reply:
I agree that there is an overhead of the codebase but this was for reasons I already mentioned - per convention no one will access property with underscore from outside, and getters provide write protection, and this is what it all is about.